lib/Federation/Proxy/TalkV1/Controller/PollController.php

```diff
@@ -171,7 +174,46 @@ public function createPoll(Room $room, Participant $participant, string $questio
 	}
 
 	/**
-	 * @return DataResponse<Http::STATUS_OK, TalkPoll, array{}>|DataResponse<Http::STATUS_ACCEPTED, null, array{}>|DataResponse<Http::STATUS_BAD_REQUEST|Http::STATUS_FORBIDDEN|Http::STATUS_NOT_FOUND, array{error: string}, array{}>
+	 * @return DataResponse<Http::STATUS_OK, TalkPollDraft, array{}>|DataResponse<Http::STATUS_BAD_REQUEST|Http::STATUS_FORBIDDEN|Http::STATUS_NOT_FOUND, array{error: 'draft'|'options'|'poll'|'question'|'room'}, array{}>
+	 * @throws CannotReachRemoteException
+	 *
+	 * 200: Draft created successfully
+	 * 201: Poll created successfully
+	 * 400: Creating poll is not possible
+	 *
+	 * @see \OCA\Talk\Controller\PollController::createPoll()
+	 */
+	public function updateDraftPoll(int $pollId, Room $room, Participant $participant, string $question, array $options, int $resultMode, int $maxVotes): DataResponse {
+		$proxy = $this->proxy->post(
```

Comment:
since this will not be part of 20.1 nor 19,
we need to check remote capabilities or add error handling here which does it the old way?
But since drafts are only moderator thing and we don't have federated moderators yet, it's actually never applicable?

Reply:
Not sure what that means - do I need to add some error handling or not? :wink:

Reply:
As discussed in the chat, since clients check the capabilites of rooms, this is not needed.